fury/interactor.py

```diff
@@ -161,6 +163,22 @@ def on_left_button_up(self, _obj, evt):
         self.selected_props["left_button"].clear()
         self.trackball_camera.OnLeftButtonUp()
 
+    def on_left_button_double_click(self, _obj, evt):
```

Comment:
this function will never be called. you should be called trough the `leftbuttondown`

Reply:
hello @skoudoro , can you explain a bit more, i did not understand.

Reply:
Understood